config/mysteriousbot.toml

```diff
@@ -1,7 +1,7 @@
 [[handlers]]
 type = "RoleWizard"
-allowed_role_grants = [ "political animal" ]
-allowed_role_revoke = [ "political animal" ]
+allowed_role_grants = [ "political animal", "Investment" ]
+allowed_role_revoke = [ "political animal", "Investment" ]
```

Comment:
That should be spelled "Investments" to be consistent with the role name in Discord.